db_diff/tskdbdiff.py

```diff
@@ -999,28 +1009,20 @@ def normalize_tsk_objects_path(guid_util: TskGuidUtils, objid: int,
                 path_parts = path_parts[:-1]
 
         if path_parts and len(path_parts) >= 2:
-            is_leapp = False
-            for leapp_module in ['aleapp', 'ileapp']:
-                if len(path_parts) > 0 and path_parts[1].lower() == leapp_module and \
-                        path_parts[-1].lower() == 'index.html':
-                    path_parts = ['ModuleOutput', leapp_module, 'index.html']
-                    is_leapp = True
+            for idx in range(0, len(path_parts) - 1):
+                if path_parts[idx].lower() == "reports" and \
+                        path_parts[idx + 1].lower().startswith("autopsytestcase html report"):
+                    path_parts = ["Reports", "AutopsyTestCase HTML Report"]
+                    break
+                if path_parts[idx].lower() == "reports" and \
+                        "html report" in path_parts[idx + 1].lower() and \
+                        len(path_parts) > idx + 2 and \
+                        path_parts[idx + 2].lower().endswith("report.html"):
+                    path_parts = ["Reports", "html-report.html"]
                     break
 
-            if not is_leapp:
-                for idx in range(0, len(path_parts) - 1):
-                    if path_parts[idx].lower() == "reports" and \
-                            path_parts[idx + 1].lower().startswith("autopsytestcase html report"):
-                        path_parts = ["Reports", "AutopsyTestCase HTML Report"]
-                        break
-                    if path_parts[idx].lower() == "reports" and \
-                            "html report" in path_parts[idx + 1].lower() and \
-                            len(path_parts) > idx + 2 and \
-                            path_parts[idx + 2].lower().endswith("report.html"):
-                        path_parts = ["Reports", "html-report.html"]
-                        break
 
-        path = os.path.join(*path_parts) if len(path_parts) > 0 else '/'
+        path = os.path.join(*path_parts) if (path_parts != None and len(path_parts)) > 0 else '/'
```

Comment:
I think the parenthesis might be off here.  Maybe something like: `path = os.path.join(*path_parts) if (path_parts is not None and len(path_parts) > 0) else '/'`?

Reply:
I think you may have missed this.